jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -520,8 +529,19 @@ def create_extract(self, add_data=False):
         inp_sp2d = self._get_ext_input_spectrum()
         trace = self._get_ext_trace()
 
-        boxcar = extract.BoxcarExtract(inp_sp2d.data, trace)
-        spectrum = boxcar(width=self.ext_width)
+        if self.ext_type_selected == 'Boxcar':
+            ext = extract.BoxcarExtract(inp_sp2d.data, trace)
+            spectrum = ext(width=self.ext_width)
+        elif self.ext_type_selected == 'Optimal':
```

Comment:
Is your choice of "Optimal" versus "Horne" in the dropdown intentional? I don't have a preference, but I wonder if others do.

Reply:
Changed to "Horne" and added links to the specreduce API docs (which should be expanded to provide more details on the individual implementations).